LanguageFeatures/Patterns/pattern_context_A05_t03.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion
+/// A simple identifier in any context named _ is treated as a wildcard variable
+/// pattern.
+///
+/// @description Checks that a simple identifier in aa assignment context named
+/// _ is treated as a wildcard variable pattern.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=patterns,records
+
+import "../../Utils/expect.dart";
+import "patterns_lib.dart";
+
+main() {
+  int _ = 0;
+  int a = 0;
+  String s = "";
+
+  (_) = 42;
+  Expect.equals(0, _);
+
+  (int _) = 42;
+  Expect.equals(0, _);
+
+  [a, _] = [1, 2];
+  Expect.equals(0, _);
+
+  [a, int _] = [1, 2];
```

Comment:
Isn't `int _` an error in an assignment context?

Reply:
Found it: https://github.com/dart-lang/language/blob/master/accepted/future-releases/0546-patterns/feature-specification.md#:~:text=It%20is%20a%20compile%2Dtime%20error%20if%20a%20variable%20pattern%20appears%20in%20an%20assignment%20context.

Reply:
Resolving: This library does not exist any more, and I can't find `[... int _]` in a declaration context any more.